### ISSUE TITLE
interacts.py error

### ISSUE THREAD

**Issue by rodrev** [bloomberg/bqplot#578]
Hi,
Just updated bqplot this morning to version 0.10.1-1.  Importing bqplot throws an AttributeError.  Here is the traceback:

```
In [1]: import bqplot
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-1-18d327bfc849> in <module>()
----> 1 import bqplot

/usr/lib/python3.6/site-packages/bqplot/__init__.py in <module>()
     58 """
     59
---> 60 from .figure import *
     61 from .axes import *
     62 from .marks import *

/usr/lib/python3.6/site-packages/bqplot/figure.py in <module>()
     35
     36 from .scales import Scale, LinearScale
---> 37 from .interacts import Interaction
     38 from .marks import Mark
     39 from .axes import Axis

/usr/lib/python3.6/site-packages/bqplot/interacts.py in <module>()
    133 @register_interaction('bqplot.PanZoom')
    134 @register
--> 135 class PanZoom(Interaction):
    136
    137     """An interaction to pan and zoom wrt scales.

/usr/lib/python3.6/site-packages/bqplot/interacts.py in wrap(interaction)
     57     def wrap(interaction):
     58         l = key if key is not None else interaction.__module__ + interaction.__name__
---> 59         interaction.types[l] = interaction
     60         return interaction
     61     return wrap

AttributeError: 'function' object has no attribute 'types'
```

**Comment by dmadeka** [comment]
@maartenbreddels This seems related to your panzoom fix. Any ideas?

**Comment by ssunkara1** [comment]
@rodrev do you mean bqplot 0.10.1? I tried it out with `0.10.1` and current master and I can't seem to reproduce the error. Will look into it a bit more.

**Comment by rodrev** [comment]
I had installed the package python-bqplot from the Arch Linux user repository.  It was working well until the package was updated to 0.10.1 (the packagers may add their own suffixes to conform to the distro's conventions).

I tried uninstalling the package and reinstalling through pip.  Apparently I had an outdated version of widgetsnbextension (2.0.0 now 3.0.8) and ipywidgets (6.0.0 now 7.0.5).

I've had trouble keeping python packages in sync before when mixing between pip and the arch package manager, hence I was avoiding pip.

Sorry for the inconvenience.  The issue was on my end.  Everything works fine now.
